tests/sles4sap/trento/setup_jumphost.pm

```diff
@@ -0,0 +1,95 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento test
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use strict;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+use registration qw(add_suseconnect_product get_addon_fullname);
+
+
```

Comment:
Is this test used in another test scenario ?

Reply:
Only called by **schedule/sles4sap/trento/trento_jumphost.yml**. It is a schedule to create a HDD image with everything installed
For the moment the test is expected to run against publiccloud_tools_xxx.qcow2

openqa-clone-job --from https://openqa.suse.de/tests/8736011 --skip-chained-deps  YAML_SCHEDULE=schedule/sles4sap/trento/trento_jumphost.yml HDD_1=publiccloud_tools_0034.qcow2  PUBLISH_HDD_1=trento_tools.qcow2

Execution example http://1c242.qa.suse.de/tests/167

Reply:
Thanks, there should be as many VRs as required to cover all code.